src/main/java/pl/wedding/management/service/WeddingHall.java

```diff
@@ -0,0 +1,200 @@
+package pl.wedding.management.service;
+
+public class WeddingHall {
+    private String address;
```

Comment:
Czy jest adres? To będzie "Polska"? Może "Polska, Warszawa"? Samo "Warszawa"? A może "ul. Knąbrnego 12"? 
Chcę przez to powiedzieć, że trzymanie zwyczajnego Stringa trzymającego adres będzie prowadziło prędzej czy później do niespójności. Trzeba to owrapować w jakąś klasę tak, żeby nie było wątpliwości jak będzie wyglądał nasz adres :)